Automattic/jetpack
# Update social previews package

Fixes https://github.com/Automattic/jetpack-reach/issues/228
Related https://github.com/Automattic/wp-calypso/pull/89483

## Proposed changes:

* Update `@automattic/social-previews` package to the latest beta

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* For a Jetpack site with Jetpack Advanced plan, create a post.
* Open Jetpack or Social sidebar and add custom media (photo/video)
* Enable "Share as a social post"
* Open Social Previews
* Goto Mastodon section
* Confirm that the "Your post" is displayed as a media post and not as link preview
* Now uncheck "Share as a social post" 
* Open Social Previews again
* Goto Mastodon section
* Confirm that the "Your post" is displayed as link preview
